mono/mini/mini-amd64.c

```diff
@@ -2870,6 +2870,10 @@ emit_call_body (MonoCompile *cfg, guint8 *code, MonoJumpInfoType patch_type, gco
 			amd64_call_code (code, 0);
 		}
 		else {
+			if (!no_patch && ((guint32)(code + 2 - cfg->native_code) % 4) != 0) {
+				guint32 pad_size = 4 - ((guint32)(code + 2 - cfg->native_code) % 4);
```

Comment:
Shouldn't we align to 8 bytes here? Aligning at 4 we still risk cross a cache line